apps/gro/views/address-outside.html

```diff
@@ -0,0 +1,54 @@
+{{<layout}}
+  {{$step}}Step 2{{/step}}
+
+  {{$journeyHeader}}
+    {{#t}}journey.header{{/t}}
+  {{/journeyHeader}}
+
+  {{$validationSummary}}
+    {{> partials-validation-summary}}
+  {{/validationSummary}}
+
+  {{$propositionHeader}}
+    {{> partials-navigation}}
+  {{/propositionHeader}}
+
+  {{$header}}{{#t}}pages.post.header{{/t}}{{/header}}
+
+  {{$content}}
+
+    {{<partials-form}}
+
+      {{$form}}
+
+        {{#input-text}}address-text-one{{/input-text}}
+
+
+        <legend class=visuallyhidden>
+          <span class="form-label-bold">{{#t}}fields.address-text-two.accessibility-label{{/t}}</span>
+        </legend>
+        {{#input-text}}address-text-two{{/input-text}}
+
+        <legend class=visuallyhidden>
+          <span class="form-label-bold">{{#t}}fields.address-text-three.accessibility-label{{/t}}</span>
+        </legend>
+        {{#input-text}}address-text-three{{/input-text}}
+
+        <legend class=visuallyhidden>
+          <span class="form-label-bold">{{#t}}fields.address-text-four.accessibility-label{{/t}}</span>
+        </legend>
+        {{#input-text}}address-text-four{{/input-text}}
+
+        <legend class=visuallyhidden>
+          <span class="form-label-bold">{{#t}}fields.address-text-five.accessibility-label{{/t}}</span>
+        </legend>
```

Comment:
hey @gxxm each field should always have a label - this isn't new functionality. If you want to hide the label in cases like this you can also pass `labelClassName: string|array` in field config - I'd put your visuallyHidden class there